### PR TITLE
Supplemental completion of almost all translated files

### PR DESCRIPTION
**Pull request by lovestoryforme**
(except for the final pdf file)

### CODE REVIEW

File: src/main/java/com/conleos/common/FormUtil.java [hunk 1]
```diff
@@ -25,10 +25,10 @@ public static Span createFormBadge(Form form) {
...
         }
         switch (form.getStatus()) {
-            case InProgress -> badge.setText("In Progress");
-            case InReview -> badge.setText("In Review");
-            case Signed -> badge.setText("Signed");
-            case Rejected -> badge.setText("Rejected");
+            case InProgress -> badge.setText(Lang.translate("view.formUtil1"));
+            case InReview -> badge.setText(Lang.translate("view.formUtil2"));
+            case Signed -> badge.setText(Lang.translate("view.formUtil3"));
+            case Rejected -> badge.setText(Lang.translate("view.formUtil4"));
```
Comment (YanSchw) [inline src/main/java/com/conleos/common/FormUtil.java:31]: view.formUtil1 is not a great placeholder for something like "In Progress"

File: src/main/java/com/conleos/common/FormUtil.java [hunk 2]
```diff
@@ -16,7 +16,7 @@ private FormUtil() {
 
     public static Span createFormBadge(Form form) {
         if (form == null || form.getStatus() == null) {
-            return new Span("Create Form");
+            return new Span(Lang.translate("view.span"));
```
Comment (YanSchw) [inline src/main/java/com/conleos/common/FormUtil.java:19]: Not a good placeholder

File: src/main/java/com/conleos/views/admin/DeleteAccountDialog.java [hunk 3]
```diff
@@ -19,25 +20,25 @@ public class DeleteAccountDialog extends Dialog {
     public DeleteAccountDialog(Long userID) {
         User account = UserService.getInstance().getUserByID(userID);
 
-        H2 changeHeader = new H2("Delete Account forever.");
-        Text text = new Text(String.format("Are you sure you want to delete '%s' ?", account.getUsername()));
-        TextField validateField = new TextField(String.format("Enter '%s' to confirm", account.getUsername()));
+        H2 changeHeader = new H2(Lang.translate("view.deleteAccount.changeHeader"));
+        Text text = new Text(String.format(Lang.translate("view.deleteAccount.text") + " '%s' ?", account.getUsername()));
+        TextField validateField = new TextField(String.format(Lang.translate("view.deleteAccount.validateField1") + " '%s' " + Lang.translate("view.deleteAccount.validateField2"), account.getUsername()));
```
Comment (YanSchw) [inline src/main/java/com/conleos/views/admin/DeleteAccountDialog.java:25]: view.deleteAccount.* should not be used.
Maybe something like 
- deleteAccountDialog.*

File: src/main/java/com/conleos/views/admin/DeleteAccountDialog.java [hunk 4]
```diff
@@ -19,25 +20,25 @@ public class DeleteAccountDialog extends Dialog {
...
         add(layout);
-        Button deleteButton = new Button("DELETE", VaadinIcon.TRASH.create());
+        Button deleteButton = new Button(Lang.translate("view.deleteAccount.button"), VaadinIcon.TRASH.create());
         deleteButton.addClickListener(clickEvent -> {
             if (account.getUsername().equals(validateField.getValue())) {
                 UserService.getInstance().deleteUserAccountByID(account.getId());
                 UI.getCurrent().getPage().reload();
             } else {
-                Notification.show("Enter Username to validate!", 4000, Notification.Position.BOTTOM_CENTER).addThemeVariants(NotificationVariant.LUMO_ERROR);
+                Notification.show(Lang.translate("view.deleteAccount.notification"), 4000, Notification.Position.BOTTOM_CENTER).addThemeVariants(NotificationVariant.LUMO_ERROR);
```
Comment (YanSchw) [inline src/main/java/com/conleos/views/admin/DeleteAccountDialog.java:35]: Not a notification

File: src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java [hunk 5]
```diff
@@ -31,7 +30,7 @@ public EditGlobalSettingsDialog() {
         dialogLayout.setWidthFull();
         add(dialogLayout);
 
-        NumberField hoursField = new NumberField("Hours In a Week");
+        NumberField hoursField = new NumberField(Lang.translate("view.editSettings.hoursField"));
```
Comment (YanSchw) [inline src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java:33]: Not part of any "View". Why is Translation part of view?

File: src/main/java/com/conleos/views/form/FormOverview.java [hunk 6]
```diff
@@ -13,23 +14,23 @@ public class FormOverview extends Tab {
...
         VerticalLayout layout = new VerticalLayout();
 
-        layout.add(addEntry(form.getMonday(), "Montag", true));
-        layout.add(addEntry(form.getTuesday(), "Dienstag", true));
-        layout.add(addEntry(form.getWednesday(), "Mittwoch", true));
-        layout.add(addEntry(form.getThursday(), "Donnerstag", true));
-        layout.add(addEntry(form.getFriday(), "Freitag", true));
-        layout.add(addEntry(form.getSaturday(), "Samstag", false));
-        layout.add(addEntry(form.getSunday(), "Sonntag", false));
+        layout.add(addEntry(form.getMonday(), Lang.translate("view.form.overview.monday"), true));
```
Comment (YanSchw) [inline src/main/java/com/conleos/views/form/FormOverview.java:17]: "Motag" is also not part of any view and the Translation does not realy on any View

File: src/main/resources/translate_en_US.properties [hunk 7]
```diff
@@ -109,12 +172,15 @@ view.profile.label.newPassword = New Password
...
 
 view.preference.pageTitle = Edit your Preferences
 view.preference.comboBox.label.language = Language
 view.preference.comboBox.label.theme = Theme
 view.preference.comboBox.label.brightTheme = Bright
 view.preference.comboBox.label.darkTheme = Dark
 
-view.instructor.notification = New reviews have been requested!
+
+view.pdf.name = YourProof.pdf
```
Comment (YanSchw) [inline src/main/resources/translate_en_US.properties:184]: this is wrong

**Review by YanSchw — CHANGES_REQUESTED** [review]

File: src/main/java/com/conleos/common/FormUtil.java (hunk 1 above)
Comment (lovestoryforme) [reply src/main/java/com/conleos/common/FormUtil.java:31]: change：
view.formUtil.progress = In Progress
view.formUtil.review = In Review
view.formUtil.sign = Signed
view.formUtil.reject = Rejected

File: src/main/java/com/conleos/common/FormUtil.java (hunk 2 above)
Comment (lovestoryforme) [reply src/main/java/com/conleos/common/FormUtil.java:19]: change：
view.span.createForm

File: src/main/java/com/conleos/views/admin/DeleteAccountDialog.java (hunk 3 above)
Comment (lovestoryforme) [reply src/main/java/com/conleos/views/admin/DeleteAccountDialog.java:25]: All deleteAccounts are replaced with deleteAccountDialog

File: src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java (hunk 5 above)
Comment (lovestoryforme) [reply src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java:33]: 微信截图_20240528222758

Comment (lovestoryforme) [reply src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java:33]: How should I change it?

File: src/main/java/com/conleos/views/form/FormOverview.java (hunk 6 above)
Comment (lovestoryforme) [reply src/main/java/com/conleos/views/form/FormOverview.java:17]: 微信截图_20240528224219

Comment (lovestoryforme) [reply src/main/java/com/conleos/views/form/FormOverview.java:17]: How should I change it?

File: src/main/java/com/conleos/views/admin/DeleteAccountDialog.java (hunk 4 above)
Comment (lovestoryforme) [reply src/main/java/com/conleos/views/admin/DeleteAccountDialog.java:35]: 微信截图_20240528224410

Comment (lovestoryforme) [reply src/main/java/com/conleos/views/admin/DeleteAccountDialog.java:35]: How should I change it?

File: src/main/resources/translate_en_US.properties (hunk 7 above)
Comment (lovestoryforme) [reply src/main/resources/translate_en_US.properties:184]: How should I change it?

File: src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java (hunk 5 above)
Comment (YanSchw) [reply src/main/java/com/conleos/views/admin/EditGlobalSettingsDialog.java:32]: it's fine leave it

**Pull request merged**